lib/widget/helpers/widgets.dart

```diff
@@ -54,7 +54,10 @@ class BoxWrapper extends StatelessWidget {
               color: boxController.backgroundColor,
               image: boxController.backgroundImage?.image,
               gradient: boxController.backgroundGradient,
-              border: boxController.borderGradient !=null
+              border: 
+              !boxController.hasBorder()?
+              null:
```

Comment:
Please keep the syntax consistent. Move the ? and : to the front of the next line.